core/unix/os.c

```diff
@@ -8765,11 +8765,24 @@ get_dynamo_library_bounds(void)
     check_start = dynamo_dll_start;
     dynamorio_libname = IF_UNIT_TEST_ELSE(UNIT_TEST_EXE_NAME, DYNAMORIO_LIBRARY_NAME);
 #    endif /* STATIC_LIBRARY */
+
+#    if !defined(STATIC_LIBRARY) && defined(LINUX)
+    static char dynamorio_libname_buf[MAXIMUM_PATH];
```

Comment:
See below: I thought we'd have a new file-scope place for the basename and replace `DYNAMORIO_LIBRARY_NAME` with it.

Reply:
(Removed)